<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SVMTrainConfig</class>
 <widget class="QWidget" name="SVMTrainConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SVM Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="svmGroupBox">
     <property name="title">
      <string>SVM</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QGridLayout" name="gridLayout_2" columnstretch="2,1">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Feature selection:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="kernelComboBox">
          <item>
           <property name="text">
            <string>rbf</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>linear</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>poly</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>sigmoid</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="cTextField">
          <property name="toolTip">
           <string>If a non-numeric value is set, the default value will be used</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>1.0</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Training data:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>C selection:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="browseTrainDataButton">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Kernel selection:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Output file name:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="outModelTextField">
          <property name="toolTip">
           <string>If no name is provided, one based on the timestamp will be used</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="featureComboBox">
          <item>
           <property name="text">
            <string>Waveform length</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="trainDataLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="progressLabel">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QLabel" name="trainAccLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Accuracy on training set:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="startTrainButton">
        <property name="text">
         <string>Start training</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
